sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultService.java

```diff
@@ -48,6 +48,10 @@
 
     private ServiceSpec serviceSpec;
 
+    public DefaultService() throws Exception {
```

Comment:
@mbdcos Why did you add a throws Exception to this method's signature? The body is empty. 

Can you please provide an example usage of this ?